README.md

```diff
@@ -35,37 +23,18 @@ To get a local copy up and running follow these simple example steps.
 
 ### Deployment
 
```

Comment:
- [ ] [OPTIONAL] Please, delete these empty sections, to make your README file clear to understand and not to be confused. But it will be better if you will add here information about how to set up your project: how to clone and etc.